src/ConfigParser.js

```diff
@@ -1,4 +1,4 @@
-const ENUM_SPLIT_REGEX = /[,\s]\s*/;
+const ENUM_SPLIT_REGEX = /\n/;
```

Comment:
Does it make sense to support `\r` here too? https://stackoverflow.com/questions/1761051/difference-between-n-and-r

Reply:
I don't think it makes sense since `\r`, by itself, is just a carriage return rather than a newline. For Windows file editing, it'd typically have `\r\n` which would still be split, and then subsequently trimmed, correctly.  I'll push a change to add `\r\n` to the test.